### PR TITLE
Update UG, DG and FindCommand message usage

### LINKED ISSUE #130: Update UG and DG for FindCommand

### PR DESCRIPTION
**Pull request by jevan13**
## Describe your changes
- Updated User Guide with the latest feature documentation
- Revised Developer Guide for alignment with current implementation
- Refined FindCommand message usage to show parameter and keywords format

## Issue ticket number and link
closes #130 
builds to #140 

## Checklist before requesting a review
- [X] I have performed a self-review of my code.
- [X] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -624,26 +624,34 @@ testers are expected to do more *exploratory* testing.
...
-   5. Test case: `find all`, `find e`, `find ph`<br>
-      Expected: Incorrect command format. Status message shows the correct usage of Find command.
-
-   6. Test case: `find ALL john`, `find E john`, `find PH john`<br>
-            Expected: Incorrect command format. Status message shows the correct usage of Find command.
-
-   7. Other incorrect find commands to try: `find aLL john`, `find pH john`, `find a`, `...`<br>
-            Expected: Similar to previous points. If the format is incorrect, the command is recognised but the action is invalid and a specific status message is shown.
+   1. Test case: `find e n/john`<br>
+   Expected: Number of employees with name `John` listed found shown in the status message.
```
Comment (thortol) [inline docs/DeveloperGuide.md:628]: Should John be in lowercase j?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -624,26 +624,34 @@ testers are expected to do more *exploratory* testing.
...
+
+   2. Test case: `find ph p/12345678`<br>
+   Expected: Number of potential hires with phone number `12345678` listed found shown in the status message. 
+   Displays the list of potential hires found.
+
+   3. Test case: `find all e/john@example.com`<br>
+   Expected: Number of employees and potential hires with email `john@example.com` listed shown in the status message.
+   Displays the list of persons found.
+   
+   4. Test case: `Find e d/IT r/SWE`<br>
```
Comment (thortol) [inline docs/DeveloperGuide.md]: Should this be find with no capital F?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -624,26 +624,34 @@ testers are expected to do more *exploratory* testing.
...
+   Expected: Number of employees with department `IT` and role `SWE` listed shown in the status message.
+   Displays the list of persons found
```
Comment (thortol) [inline docs/DeveloperGuide.md]: Should this display the list of employees found?

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 4]
```diff
@@ -22,9 +27,16 @@ public class FindCommand extends Command {
     public static final String MESSAGE_PURPOSE = "Finds all employees and/or potential hires whose names "
             + "contain any of the specified keywords (case-insensitive) and displays them as a list with "
             + "index numbers.";
-    public static final String MESSAGE_FORMAT = COMMAND_WORD + " (" + FindEmployeeCommand.ARGUMENT_WORD
-        + "/" + FindPotentialCommand.ARGUMENT_WORD + "/" + FindCommand.ARGUMENT_WORD + ") KEYWORD [MORE_KEYWORDS]";
-    public static final String MESSAGE_EXAMPLE = COMMAND_WORD + " " + ARGUMENT_WORD + " n/alice bob e/alice@gmail.com";
+    public static final String COMMAND_PARAMETERS = FindEmployeeCommand.ARGUMENT_WORD + " for employees, "
+            + FindPotentialCommand.ARGUMENT_WORD + " for potential hires, and "
+            + FindCommand.ARGUMENT_WORD + " for both employees and potential hires.";
+    public static final String COMMAND_KEYWORDS = PREFIX_NAME + "/[NAME] " + PREFIX_PHONE + "/[PHONE NUMBERS] "
```
Comment (thortol) [inline src/main/java/seedu/address/logic/commands/FindCommand.java]: Should NAME be NAMES instead?
Also can you use PHONE_NUMBERS instead as it follows the same format as edit?

**Review by thortol — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jevan13) [reply docs/DeveloperGuide.md:628]: Yes! Thank you. Fixed.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (jevan13) [reply docs/DeveloperGuide.md]: Yes! Thank you. Fixed.

**Review by thortol — APPROVED** [review]

**Pull request merged**